docs/src/main/sphinx/udf/function.md

```diff
@@ -58,6 +59,10 @@ The `COMMENT` characteristic can be used to provide information about the
 function to other users as `description`. The information is accessible with
 [](/sql/show-functions).
 
+Use the `WITH properties AS block` section to define the `handler` property for
+a [](/udf/python) to specify the Python method to invoke as the function. The
+Python code in `block` is encapsulated within `$$`.
+
 The body of the UDF can either be a simple single `RETURN` statement with an
```

Comment:
This needs to be split to account for the different syntaxes of SQL vs other languages. Maybe something like

> For SQL UDFs, the body can be ...

> For UDFs in other languages, the definition is a dollar-quoted string ...